db/1.9/modules/ROOT/pages/database-connector-transactions.adoc

```diff
@@ -0,0 +1,78 @@
+= Configure Database Transactions Example - Mule 4
+
+Anypoint Connector for Database (Database Connector) enables you to execute database operations in the context of a transaction. Each operation has a Transactional Action field where you specify the type of joining action that operations can take regarding the active transaction.
+
+The following examples illustrates how to configure the Transactional Action field in Studio both for the Select operation, and when grouping operations using the Try scope component.
```

Comment:
GLOBAL in these topics: replace "examples illustrates" with either
example illustrates
examples illustrate
(noun-verb agreement)
If this is in the template, it needs to be fixed.

Reply:
commas:
```suggestion
The following examples illustrates how to configure the Transactional Action field in Studio, both for the Select operation and when grouping operations using the Try scope component.
```
Is the name of the component actually Try Scope?